packages/components/src/internal/components/editable/Cell.tsx

```diff
@@ -93,17 +96,15 @@ export class Cell extends React.PureComponent<Props> {
             ],
             MODIFICATION_TYPES.REPLACE
         );
-        if (onCellModify) onCellModify();
```

Comment:
it looks like two usages in LKFM (`AddSampleView.tsx` and`MoveSampleView.tsx`). I didn't look too closely at those usages. Maybe they wouldn't be needed anymore after this refactor. Or is this `onCellModify` called elsewhere now?

Reply:
Those use EditableGridPanelDeprecated, which still calls that callback. In the future it is not needed, this type of handler can be emulated via onGridChange (basically if the EditorModel cellValues changed then a cell was modified).